src/cgame/cg_rocket_draw.cpp

```diff
@@ -921,22 +921,36 @@ class PositionElement : public TextHudElement
 {
 public:
 	PositionElement( const Rml::String& tag ) :
-			TextHudElement( tag, ELEMENT_GAME, true )
+			TextHudElement( tag, ELEMENT_GAME, true ),
+			lastpos_(),
+			shown_(false)
 	{
 	}
 
 	void DoOnUpdate() override
 	{
 		if ( !cg_drawPosition.Get() )
 		{
-			SetText( "" );
+			if ( shown_ )
+			{
+				SetText( "" );
+				shown_ = false;
+			}
 			return;
 		}
 
-		// Add text to be configured via CSS
-		vec3_t const & origin = cg.predictedPlayerState.origin;
-		SetText( va( "%0.0f %0.0f %0.0f", origin[0], origin[1], origin[2] ) );
+		shown_ = true;
+		glm::vec3 origin = VEC2GLM( cg.predictedPlayerState.origin );
+		if ( lastpos_ != origin )
+		{
+			SetText( Str::Format( "%0.0f %0.0f %0.0f",
```

Comment:
Use %g instead?

Reply:
The idea was to use integer rounding

Reply:
integer rounding also makes it possible to have more stable display as in https://github.com/UnvanquishedAssets/unvanquished_src.dpkdir/pull/154